api-schema/src/main/java/org/stellar/anchor/api/sep/sep12/PutCustomerVerificationResponse.java

```diff
@@ -1,10 +1,12 @@
 package org.stellar.anchor.api.sep.sep12;
 
+import org.stellar.anchor.api.callback.GetCustomerResponse;
+
 /**
  * The response to the PUT /customer/verification endpoint of SEP-12.
  *
  * @see <a
  *     href="https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#fields">Refer
  *     to SEP-12</a>
  */
-public class PutCustomerVerificationResponse extends Sep12GetCustomerResponse {}
+public class PutCustomerVerificationResponse extends GetCustomerResponse {}
```

Comment:
We don't need to refactor this right now, but this is a little strange to read.

Reply:
It is a bug to extend from Sep12GetCustomerResponse.